src/status_im/ui/screens/hardwallet/pin/styles.cljs

```diff
@@ -21,10 +21,10 @@
   {:color      colors/red
    :text-align :center})
 
-(def center-container
+(defn center-container [title]
```

Comment:
Makes sense to rename it to `title?`

Reply:
@siphiuel I don't have strong preference on how to name styles def/defn, i prefer to inline it into views so no naming required. 

`hardwallet/pin/styles` and `hardwallet/pin/views` will be removed anyway once PIN view from basic onboarding will be ready, i'll be using that version for unification.

Reply:
Sorry for confusion - i was nitpicking about argument argument (`title` -> `title?`), because it's used as a boolean, but now checked that the actual title is passed to `center-container`, so this renaming does not make sense.